enterprise/backend/src/metabase_enterprise/serialization/load.clj

```diff
@@ -195,19 +195,19 @@
   (.getName (io/file path)))
 
 (defn- unresolved-names->string
-  ([entity]
-   (unresolved-names->string entity nil))
-  ([entity insert-id]
+  ([model]
+   (unresolved-names->string model nil))
+  ([model insert-id]
```

Comment:
A lot of the SerDes v2 code uses `entity` to mean the actual row/object getting serialized or deserialized, so this was pretty confusing. I changed it to `model` to make it a little clearer that it's referring to the actual Toucan model e.g. `User`